CareKitStore/CareKitStore/Protocols/CarePlans/OCKAnyCarePlanStore.swift

```diff
@@ -138,7 +138,7 @@ public extension OCKAnyReadOnlyCarePlanStore {
                           completion: @escaping OCKResultClosure<OCKAnyCarePlan>) {
         var query = OCKCarePlanQuery(for: Date())
         query.limit = 1
-        query.sortDescriptors = [.effectiveDate(ascending: true)]
+        query.sortDescriptors = [.effectiveDate(ascending: false)]
```

Comment:
All of these should be false to always get the newest version related to today, similar to how the fetching task was set up.